omod/src/main/web/dashboardwidgets/latestobsforconceptlist/latestobsforconceptlist.controller.js

```diff
@@ -15,39 +14,35 @@ export default class LatestObsForConceptListController {
         this.maxAgeInDays = this.widgetsCommons.maxAgeToDays(this.config.maxAge);
 
         // Remove whitespaces
-        this.config.concepts = this.config.concepts.replace(/\s/g,'');
-
-        let concepts = this.config.concepts.split(",");
-        for (let i = 0; i < concepts.length; i++) {
-            let concept = concepts[i];
-            // Fetch last obs for concept
```

Comment:
Is this just a formatting change? It is hard to tell what exactly changed.

Reply:
@dkayiwa The whole loop changed, removed an outer loop and how it works, so instead of fetching one concept at a time, we process the results for multiple concepts at a go